File: CSharp/RetrieveTheSubreddit.cs
namespace CSharp
{
    // Create a function to extract the name of the subreddit from its URL.
    // https://edabit.com/challenge/qvo2Ae8WdRQfCwaQk
    public static class RetrieveTheSubreddit
    {
        public static string SubReddit(string link)
        {
            var connUri = link.Replace("https://", string.Empty);

            return connUri.Split('/')[2];
        }
    }
}

File: js/retrieveTheSubreddit.js
// Create a function to extract the name of the subreddit from its URL.
// https://edabit.com/challenge/WT5MGmgaP3jvx8qpt

const subReddit = (link) => {
  let connUri = link.replace("https://", "");

  return connUri.split("/")[2];
};

module.exports = subReddit;
